I get problems with predicting between torch, cuda9.2, and python3. Any ideas?
```
(iceberg_seals) [bspitz@gpu047 Seals]$ iceberg_seals.predicting --input_image=WV03_20171122043753_104001003677D100_17NOV22043753-P1BS-501956508070_01_P003_u08rf3031.tif --model_architecture=UnetCntWRN --hyperparameter_set=A --training_set=test_vanilla --test_folder=./test --model_path=./UnetCntWRN/ --output_folder=./test_image
Traceback (most recent call last):
  File "/pylon5/mc3bggp/bspitz/Seals/iceberg_seals/bin/iceberg_seals.predicting", line 14, in <module>
    import torch
  File "/pylon5/mc3bggp/bspitz/Seals/iceberg_seals/lib/python3.5/site-packages/torch/__init__.py", line 81, in <module>
    from torch._C import *
ImportError: numpy.core.multiarray failed to import
(iceberg_seals) [bspitz@gpu047 Seals]$ module load torch
torch/5633c24e(22):ERROR:151: Module 'torch/5633c24e' depends on one of the module(s) 'python/2.7.11_gcc_np1.11'
torch/5633c24e(22):ERROR:102: Tcl command execution failed: prereq python/2.7.11_gcc_np1.11
```

--- a/README.md
+++ b/README.md
@@ -22,27 +22,27 @@
 ### PSC Bridges
 From source:
 ```bash
-iparask$ git clone https://github.com/iceberg-project/Seals.git
-iparask$ module load cuda
-iparask$ module load python3
-iparask$ virtualenv iceberg_seals
-iparask$ source iceberg_seals/bin/activate
-[iceberg_seals] iparask$ export PYTHONPATH= <path>/iceberg_seals/bin/activate
-[iceberg_seals] iparask$ pip install . --upgrade
+$ git clone https://github.com/iceberg-project/Seals.git
+$ module load cuda
+$ module load python3
+$ virtualenv iceberg_seals
+$ source iceberg_seals/bin/activate
+[iceberg_seals] $ export PYTHONPATH= <path>/iceberg_seals/bin/activate
+[iceberg_seals] $ pip install . --upgrade
 ```
 
 From PyPi:
 ```bash
-iparask$ module load cuda
-iparask$ module load python3
-iparask$ virtualenv iceberg_seals
-iparask$ source iceberg_seals/bin/activate
-[iceberg_seals] iparask$ export PYTHONPATH= <path>/iceberg_seals/bin/activate
-[iceberg_seals] iparask$ pip install iceberg_seals.search
+$ module load cuda
+$ module load python3
+$ virtualenv iceberg_seals
+$ source iceberg_seals/bin/activate
+[iceberg_seals] $ export PYTHONPATH= <path>/iceberg_seals/bin/activate
+[iceberg_seals] $ pip install iceberg_seals.search
 ```
 
 To test
 ```bash
-[iceberg_seals] iparask$ iceberg_seals.tiling --scale_bands=299 --input_image=<image_abspath> --output_folder=./test
-[iceberg_seals] iparask$ iceberg_seals.predicting --input_image=<image_filename> --model_architecture=UnetCntWRN --hyperparameter_set=A --training_set=test_vanilla --test_folder=./test --model_path=./ --output_folder=./test_image
+[iceberg_seals] $ iceberg_seals.tiling --scale_bands=299 --input_image=<image_abspath> --output_folder=./test
+[iceberg_seals] $ iceberg_seals.predicting --input_image=<image_filename> --model_architecture=UnetCntWRN --hyperparameter_set=A --training_set=test_vanilla --test_folder=./test --model_path=./ --output_folder=./test_image
 ```

